### ISSUE TITLE
Having "Nimator" as both a class and namespace creates confusion

### ISSUE THREAD

**Issue by jeroenheijmans** [omniaretail/Nimator#16]
The `Nimator.ExampleConsoleApp` has this example code:

```csharp
var json = reader.ReadToEnd();
return Nimator.FromSettings(logger, json);
```

However, if you create a new fresh console application and reference Nimator, and copy-paste said code to your code file, you'll not be able to compile. Instead, you need to do this:

```csharp
var json = reader.ReadToEnd();
return Nimator.Nimator.FromSettings(logger, json); // <-- Subtly different! "Nimator.Nimator..."
```

This is caused by the fact that the default namespace for the _example_ app is part of a sub-namespace (`Nimator.ExampleConsoleApp`) of the `Nimator` namespace, whereas the fresh console app isn't.

Bottom line: it's confusing to have both a `class` _and_ a `namespace` called "`Nimator`". Please consider changing this setup.
